chi/server.py

```diff
@@ -339,13 +339,13 @@ def show_flavor_by_name(name) -> NovaFlavor:
 ##########
 
 def get_server(ref) -> NovaServer:
-    """Get a server by its ID or name.
+    """Get a server by its ID.
```

Comment:
Why not support lookup by name? And the function didn't actually change, so the documentation isn't correct, it would seem.

Reply:
`get_server(ref)` points to `show_server(ref)`, and neither of these functions point to `show_server_by_name(ref)` which would need to be called to support lookup by name. Currently, the only way to obtain a server via its name is deliberately through `show_server_by_name(ref)`.

Reply:
OK, I'll look in to that separately then :)